### ISSUE TITLE
Community.login cookie property missing

### ISSUE THREAD

**Issue by endze1t** [DoctorMcKay/node-steamcommunity#321]
Hey there,

since the last Steam maintenance, a few bot accounts (5 out of 2,500) have not received the `steamLogin` / `steamLoginSecure` cookie property when using `Community.login()`.

Options:
```js
{
    accountName,
    password,
    twoFactorCode,
    disableMobile: true,
  }
  ``` 

Only receiving the following properties:
```js
[
  'Steam_Language=english',
  'timezoneOffset=0,0',
  'steamCountry=DE%7C.....',
  'sessionid=asdf1234.....'
]
``` 

With these cookies, other libraries like `tradeManager.setCookies()` will fail because of `Error: Not Logged In`.

I'm really clueless, all bots using the same codebase, and only 5 of 2,500 bots are no longer working. Manually logging into these accounts is working. Also, the bots receiving necessaries cookies on other libraries like `node-steam-client`, only `node-steamcommunity` having these cookie issues right now.

The bots tested on different IP addresses and made a cooldown over 6 hours.

**Comment by DoctorMcKay** [comment]
Unfortunately, I don't think I'm going to be able to do anything about an issue that I can't reproduce, and it doesn't seem that I'll be able to reproduce it if it's such a rare issue.

You may want to try logging in using [steam-session](https://www.npmjs.com/package/steam-session) instead, which is a more modern way of authenticating. I'm working on steamcommunity v4 which will include an auth overhaul, but I can't say when it'll be ready. Alternatively, if you're already using steam-user, that's a better way to get your cookies.

**Comment by Rodrigo-JM** [comment]
@endze1t have you checked if you're using the latest versions of each lib?

**Comment by DoctorMcKay** [comment]
Looks like this means of logging in doesn't work anymore. It'll be fixed in steamcommunity v4, but until then you can use [steam-session](https://www.npmjs.com/package/steam-session) or [steam-user](https://www.npmjs.com/package/steam-user) to get your login cookies.

**Comment by Shybko** [comment]
Hi.

Yesterday all of my bots stopped login to the steam. Nothing was changed. Looks like this global problem.

Here is some log from my console

```
info: the bot (xan....) try login steam... wait
info: the bot logged in, steamID: [U:1:148.....], Login: (xan....)
info: got the command for bot # xan.... doSyncInventory at 1697706209739
Error: Not Logged In
    at TradeOfferManager.getInventoryContents (/bot/node_modules/steam-tradeoffer-manager/lib/index.js:292:12)
    at Bot.getInventoryContents (/bot/src/modules/bot/Bot.js:493:35)
    at retryAttempt (/bot/node_modules/async/dist/async.js:4614:9)
    at Object.retry (/bot/node_modules/async/dist/async.js:4625:5)
    at Function.withRetry (/bot/src/modules/bot/Bot.js:574:13)
    at Bot.doSyncInventory (/bot/src/modules/bot/Bot.js:757:11)
    at /bot/src/index.js:45:36
    at Channel.BaseChannel.dispatchMessage (/bot/node_modules/amqplib/lib/channel.js:484:12)
    at Channel.BaseChannel.handleDelivery (/bot/node_modules/amqplib/lib/channel.js:493:15)
    at Channel.emit (events.js:315:20)
    at /bot/node_modules/amqplib/lib/channel.js:279:10
    at Channel.content [as handleMessage] (/bot/node_modules/amqplib/lib/channel.js:332:9)
    at Channel.C.acceptMessageFrame (/bot/node_modules/amqplib/lib/channel.js:247:31)
    at Channel.C.accept (/bot/node_modules/amqplib/lib/channel.js:400:17)
    at Connection.mainAccept [as accept] (/bot/node_modules/amqplib/lib/connection.js:64:33)
    at Socket.go (/bot/node_modules/amqplib/lib/connection.js:478:48)
debug: errorFilter debug: errCode: Not Logged In
error: getInventoryContexts 570 error
error:
{
  message: 'Not Logged In',
  stack: 'Error: Not Logged In\n' +
    '    at TradeOfferManager.getInventoryContents (/bot/node_modules/steam-tradeoffer-manager/lib/index.js:292:12)\n' +
    '    at Bot.getInventoryContents (/bot/src/modules/bot/Bot.js:493:35)\n' +
    '    at retryAttempt (/bot/node_modules/async/dist/async.js:4614:9)\n' +
    '    at Object.retry (/bot/node_modules/async/dist/async.js:4625:5)\n' +
    '    at Function.withRetry (/bot/src/modules/bot/Bot.js:574:13)\n' +
    '    at Bot.doSyncInventory (/bot/src/modules/bot/Bot.js:757:11)\n' +
    '    at /bot/src/index.js:45:36\n' +
    '    at Channel.BaseChannel.dispatchMessage (/bot/node_modules/amqplib/lib/channel.js:484:12)\n' +
    '    at Channel.BaseChannel.handleDelivery (/bot/node_modules/amqplib/lib/channel.js:493:15)\n' +
    '    at Channel.emit (events.js:315:20)\n' +
    '    at /bot/node_modules/amqplib/lib/channel.js:279:10\n' +
    '    at Channel.content [as handleMessage] (/bot/node_modules/amqplib/lib/channel.js:332:9)\n' +
    '    at Channel.C.acceptMessageFrame (/bot/node_modules/amqplib/lib/channel.js:247:31)\n' +
    '    at Channel.C.accept (/bot/node_modules/amqplib/lib/channel.js:400:17)\n' +
    '    at Connection.mainAccept [as accept] (/bot/node_modules/amqplib/lib/connection.js:64:33)\n' +
    '    at Socket.go (/bot/node_modules/amqplib/lib/connection.js:478:48)'
}
```

**Comment by DoctorMcKay** [comment]
You'll need steam-user 4.29.0 (released July 2023) or later for web authentication to work.

**Comment by DoctorMcKay** [comment]
This is more of a workaround than a fix, but [v3.47.1](https://github.com/DoctorMcKay/node-steamcommunity/releases/tag/v3.47.1) does fix the login() method.

**Comment by g0dzcsgo** [comment]
> This is more of a workaround than a fix, but [v3.47.1](https://github.com/DoctorMcKay/node-steamcommunity/releases/tag/v3.47.1) does fix the login() method.

Yes but that means you can\t use proxies

**Comment by DoctorMcKay** [comment]
> > This is more of a workaround than a fix, but [v3.47.1](https://github.com/DoctorMcKay/node-steamcommunity/releases/tag/v3.47.1) does fix the login() method.
> 
> Yes but that means you can\t use proxies

Correct, which is why I say it's more of a workaround than a fix. If you need to use a proxy, use steam-session or steam-user.

**Comment by g0dzcsgo** [comment]
> > Yes but that means you can\t use proxies
> 
> Correct, which is why I say it's more of a workaround than a fix. If you need to use a proxy, use steam-session or steam-user.

I tried this, it still gives me an error saying:  You must be logged in to perform that action

When using the cookies and session from steam-session, aswell as if I try with steam-user same result. I posted about it on the forums and I included my code.
